softwareComponents/simplesimLib/src/locked_module_communication.cpp

```diff
@@ -18,6 +18,7 @@ LockedModuleCommunication::LockedModuleCommunication( CommandHandler & commandHa
                                 &LockedModuleCommunication::onRofiCmd,
                                 this ) )
 {
+    _logger.logSubscribe( "~/" + moduleTopicName + "/control" );
```

Comment:
Add the log below the asserts - they are at the top so that it describes the contract.